### DISCUSSION TITLE
novo no ubuntu

### DISCUSSION THREAD

**Discussion by HigorAln** [frontendbr/forum#2165] [Perguntas e Respostas]
Olá devs tudo bem?...

bem recentemente instalei o sistema operacional ubuntu para codar e experimentar algo novo, gostaria de uns conselhos de algo de interessante para instalar que seja um auxiliar para o visual code, ou no terminal, ou algo no ubuntu que possa melhorar a performance... em fim, gostaria de algumas dicas... agradeço desde já... :D

**Comment by jpferreiradev** [comment]
Vai no canal no youtube do DioLinux, lá tem bastante coisa massa que ele mostra e ensina...

**Reply by HigorAln** [reply]
valeuuu!
